backend/src/schema/resolvers/follow.spec.js

```diff
@@ -17,20 +24,42 @@ const mutationUnfollowUser = id => `
   }
 `
 
-beforeEach(async () => {
-  await factory.create('User', {
-    id: 'u1',
-    email: 'test@example.org',
-    password: '1234',
-  })
-  await factory.create('User', {
-    id: 'u2',
-    email: 'test2@example.org',
-    password: '1234',
+beforeAll(() => {
```

Comment:
FYI @vbelolapotkov the `beforeAll` in most of our backend tests is *not only* a performance thing. There is also a bug in [graphql middlewares](https://github.com/prisma/graphql-middleware/issues/63#issuecomment-520786385) which will wrap another layer of middlewares around each resolver for every test case if you use `beforeEach`.

Reply:
Good point, thanks for the info